code/controllers/subsystem/pai.dm

```diff
@@ -3,16 +3,25 @@ SUBSYSTEM_DEF(pai)
 
 	flags = SS_NO_INIT|SS_NO_FIRE
 
+	/// List of pAI candidates, including those not submitted.
 	var/list/candidates = list()
+	/// Prevents a crew member from hitting "request pAI"
 	var/request_spam = FALSE
+	/// Prevents a pAI from submitting itself repeatedly and sounding an alert.
 	var/submit_spam = FALSE
+	/// All pAI cards on the map.
 	var/list/pai_card_list = list()
 
 /datum/pai_candidate
```

Comment:
Document this as well, such as where in the code its used or you can acquire one.